### PR TITLE
feat(nimbus): check Remote Settings connectivity in heartbeat

### PR DESCRIPTION
**Pull request by leplatrem**
Because

- We want to discover Remote Settings connectivity issue before the background tasks fail

This commit

- Registers a new check
- Checks heartbeat
- Checks credentials

### CODE REVIEW

**Review by ahoneiser — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -34,13 +34,13 @@ RUFF_CHECK = ruff experimenter/ tests/
...
-LOAD_LANGUAGES = python manage.py loaddata ./experimenter/base/fixtures/languages.json
-LOAD_FEATURES = python manage.py load_feature_configs
-LOAD_DUMMY_EXPERIMENTS = [[ -z $$SKIP_DUMMY ]] && python manage.py load_dummy_experiments || python manage.py load_dummy_projects
+CHECK_DOCS = python manage.py generate_docs --check=true --skip-checks
+GENERATE_DOCS = python manage.py generate_docs --skip-checks
+LOAD_COUNTRIES = python manage.py loaddata ./experimenter/base/fixtures/countries.json --skip-checks
+LOAD_LOCALES = python manage.py loaddata ./experimenter/base/fixtures/locales.json --skip-checks
+LOAD_LANGUAGES = python manage.py loaddata ./experimenter/base/fixtures/languages.json --skip-checks
+LOAD_FEATURES = python manage.py load_feature_configs --skip-checks
+LOAD_DUMMY_EXPERIMENTS = [[ -z $$SKIP_DUMMY ]] && python manage.py load_dummy_experiments --skip-checks || python manage.py load_dummy_projects --skip-checks
```
Comment (yashikakhurana) [inline Makefile:43]: I am just wondering why `--skip-checks` flag?

**Review by yashikakhurana — COMMENTED** [review]
I am curious to know about the addition of `--skip-checks` 🤔

**Comment by yashikakhurana** [comment]
> > I am curious to know about the addition of --skip-checks 🤔
> 
> Unfortunately, the checks are executed implicitly on most `manage.py` commands, including migrations, loaddata, etc.
> 
> Since python-dockerflow uses this check framework for the `__heartbeat__` endpoint, we have to adapt :/ I could not find any other way to register the check only once the application has truely started to listen (via runserver or wsgi)

Got your point, my only concern skipping checks can lead to unexpected behavior or errors 👀, lets see what @jaredlockhart thoughts are on this

**Review by jaredlockhart — APPROVED** [review]
Awesome thanks so much @leplatrem 🙏 This will be very handy.

As far as skipping the checks, yes @leplatrem is right, all management commands invoke them and it's not reasonable to expect an rs instance is always availble whenever they're run.  Those commands should still fail/succeed for the specific things they're doing/checking for regardless of whether the checks are in place. cc @yashikakhurana 

I'll merge this 👍

**Comment by yashikakhurana** [comment]
@jaredlockhart perfect, makes sense, thank you :)

**Pull request merged**